cleverhans/devtools/tests/test_format.py

```diff
@@ -51,8 +84,12 @@ def print_files_information_pep8():
   list of files which do not respect PEP8 formatting that aren't in the
   whitelist
   """
+
+  raise NotImplementedError("Broken by move to TensorFlow style")
+
```

Comment:
Why are we keeping what's below if it is broken?

Reply:
A lot of it will be useful for implementing the pylint version